openduck-py/openduck_py/routers/voice.py

```diff
@@ -396,6 +399,25 @@ async def websocket_consumer(queue: asyncio.Queue, websocket: WebSocket):
             queue.task_done()
 
 
+@audio_router.post("/start")
+async def create_room_and_start():
+    room_info = await create_room()
+    print("created room")
+    import multiprocessing
+
+    process = multiprocessing.Process(
+        target=run_connect_daily, args=(room_info["url"],)
+    )
+    process.start()
```

Comment:
Will the processes get killed once the meeting ends?

Reply:
Now it will shut down when the last human leaves: [2bc0b48](https://github.com/uberduck-ai/openduck/pull/60/commits/2bc0b480d5c15de4c405192e3d0eeee306fb0d0c)